app/workers/find_a_candidate/pool_invite_chaser_worker.rb

```diff
@@ -0,0 +1,26 @@
+module FindACandidate
+  class PoolInviteChaserWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :default
+
+    def perform
+      invites = Pool::Invite.current_cycle.published.not_responded
+        .where.missing(:chasers_sent)
+        .where(
+          application_form_id: Pool::Invite.current_cycle.published.not_responded
+            .where.missing(:chasers_sent)
+            .where('sent_to_candidate_at <= ? ', 1.day.ago)
```

Comment:
Chaser should be sent after 24 hours

Reply:
I might be losing my mind, but doesn't this say sent_to_candidate_at is before 1.day.ago? Should it be:
```suggestion
            .where('sent_to_candidate_at > ? ', 1.day.ago)
```

Reply:
Ok, thanks, I have clearly been on holiday and cannot think clearly.